src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -625,6 +627,25 @@ class GmailThreadView {
     return view;
   }
 
+  addSubjectButton(button: any) {
+    const subjectToolbarElement = this._findSubjectToolbarElement();
+    if (!subjectToolbarElement) {
+      throw new Error('Subject toolbar element not found');
+    }
+    // gmail has these spacer spans. Not sure if important or a good idea to use this way
+    const el = document.createElement('span');
+    el.setAttribute('data-unique-tt-id', 'ucc-2');
```

Comment:
oh yeah and it shouldn't be hardcoded to 2 if this is actually kept in here. should be consecutive like the gmail ones are